api/envoy/extensions/access_loggers/filters/local_ratelimit/v3/local_ratelimit.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.filters.local_ratelimit.v3;
+
+import "envoy/config/core/v3/config_source.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.filters.local_ratelimit.v3";
+option java_outer_classname = "LocalRatelimitProto";
+option java_multiple_files = true;
+option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/access_loggers/filters/local_ratelimit/v3;local_ratelimitv3";
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: LocalRateLimitFilter]
+// [#extension: envoy.access_loggers.extension_filters.local_ratelimit]
+
+// Filters for rate limiting the access log emission using global token buckets per process and shared across all listeners.
+message LocalRateLimitFilter {
+  // Specify the :ref:`name <envoy_v3_api_field_type.v3.TokenBucketConfig.name>` for lookup.
+  string resource_name = 1 [(validate.rules).string = {min_len: 1}];
+
+  // The configuration source for the :ref:`token_bucket <envoy_v3_api_field_type.v3.TokenBucketConfig.token_bucket>`.
+  // It should stay the same through the process lifetime.
+  // TODO(taoxuy): add support for dynamic config_source if needed.
```

Comment:
This should not be mentioned in the public docs, it belongs in the code. You can open an issue for tracking it.

Reply:
removed.